plugins/GSdx/GSPng.cpp

```diff
@@ -20,15 +20,59 @@
 
 #include "stdafx.h"
 #include "GSPng.h"
+#include <png.h>
 
 namespace GSPng {
 
+    void SaveFile(const string& file, uint8** image, uint32 width,
+            uint32 height, int channel_bit_depth, int type, bool rb_swapped = false)
+    {
+        FILE *fp = fopen(file.c_str(), "wb");
+        if (fp == nullptr)
+            return;
+
+        png_structp png_ptr = png_create_write_struct(PNG_LIBPNG_VER_STRING,
+                nullptr, nullptr, nullptr);
+        if (png_ptr == nullptr) {
+            fclose(fp);
+            return;
+        }
+
+        png_infop info_ptr = png_create_info_struct(png_ptr);
+        if (info_ptr == nullptr) {
+            png_destroy_write_struct(&png_ptr, nullptr);
+            fclose(fp);
+            return;
+        }
+
+        if (setjmp(png_jmpbuf(png_ptr))) {
+            png_destroy_write_struct(&png_ptr, nullptr);
+            fclose(fp);
+            return;
+        }
+
```

Comment:
> Because this file is this file, the following file is the following file? ;)

I actually meant _[this file](https://github.com/turtleli/pcsx2/blob/gsdx-png/plugins/GSdx/GSPng.cpp)_ when I said _``the following file``_. (**not the succeeding file**)

> If I did that, the style in this file would be inconsistent..

only the braces for the namespace seems to be used on the same line at the file. all other braces ( for the functions) are used on a new line.

>  Also, I prefer my braces on the same line

Hmm. If you don't mind - can you explain why you prefer having it that way ? Personally I like having the braces on a new line since it makes both of the braces aligned and easier to be spotted.